src/pyaml_env/parse_config.py

```diff
@@ -1,15 +1,15 @@
-import os
-import re
 import yaml
+from .constructor import PyamlEnvConstructor
```

Comment:
Could you please use absolute imports? :)

Reply:
done